### PR TITLE
Add argument to /skull to give skull to other player

### LINKED ISSUE #5762: Add an option to give someone their skull using /skull [player/skin] [player]
### Feature description

The ability to give a player their own skull using EssentialsX

### How the feature is useful

This simply extends the exisiting /skull command so admins can give heads to other players.

### PR DESCRIPTION
**Pull request by Yurakaii**
### Information

This PR closes #5762. 

### Details
**Proposed feature:**  
This adds the ability to give a skull to another player using the /skull command with a second argument, if you only provide one argument the command will work as it previously did.

The new usage of the skull command is:
`/<command> [owner] [player]`

**Environments tested:**    

OS: Windows 11

Java version:  21

- [x] Most recent Paper version (1.20.6, git-Paper-137)
- [ ] CraftBukkit/Spigot/Paper 1.12.2
- [ ] CraftBukkit 1.8.8

**Demonstration:**  

This gives the dinnerbone's skull to trashdrumlinfly:
Screenshot 2024-06-05 144150

trashdrumlinfly received dinnerbone's skull
Screenshot 2024-06-05 143929

The command can still be used as how it worked before by just providing the first argument.

### CODE REVIEW

**Review by JRoy — CHANGES_REQUESTED** [review]
overall looks fine, can you just update the `skullCommandUsage` messages in the `messages.properties` file

**Comment by CatTeaA** [comment]
bump

**Comment by Huge-mistake** [comment]
> overall looks fine, can you just update the `skullCommandUsage` messages in the `messages.properties` file

Apologies for the interruption, can this PR be merged now? I would like to utilize this feature in the menu to provide players with skulls.

File: Essentials/src/main/java/com/earth2me/essentials/commands/Commandskull.java [hunk 1]
```diff
@@ -48,7 +48,14 @@ public Commandskull() {
     @Override
     protected void run(final Server server, final User user, final String commandLabel, final String[] args) throws Exception {
         final String owner;
-        if (args.length > 0 && user.isAuthorized("essentials.skull.others")) {
+        final User player;
+        if(args.length == 2){
```
Comment (JRoy) [inline Essentials/src/main/java/com/earth2me/essentials/commands/Commandskull.java]:
```suggestion
        if (args.length == 2) {
```

**Review by JRoy — APPROVED** [review]

**Comment by JRoy** [comment]
Thanks for the contribution :)

**Pull request merged**